LoginPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
+        window.location.href = '/dashboard';
+      } else {
```

Comment:
Avoid using alert for error handling. It blocks the entire browser and the user interaction until it's closed. Consider using a less intrusive method for error handling, like a toast notification or a custom error component.